oeps/oep-38-Data-Modeling.rst

```diff
@@ -65,39 +46,39 @@ This OEP explicitly does not cover:
 Definitions
 ===========
 
-BI - Business intelligence. Technologies, applications and practices for
+**BI** - Business intelligence. Technologies, applications and practices for
 the collection, integration, analysis, and presentation of business
 information. The purpose of Business Intelligence is to support better
 business decision making.
 
-Composite Key - in the context of relational databases, a composite key
+**Composite Key** - in the context of relational databases, a composite key
 is a combination of two or more columns in a table that can be used to
 uniquely identify each row in the table. Uniqueness is only guaranteed
 when the columns are combined; when taken individually, the columns do
 not guarantee uniqueness.
 
-CUD - Create, Update, Delete. These are some of the actions that may
+**CUD** - Create, Update, Delete. These are some of the actions that may
 happen to data in a database system.
 
-Data Best Practices - These practices are designed to help teams create
+**Data Best Practices** - These practices are designed to help teams create
 rich and efficient data models within the OpenedX ecosystem. They are not
 standards but guidelines to help teams think about how to store data.
 
-Data Dimension - A Data Dimension is a set of data attributes pertaining
+**Data Dimension** - A Data Dimension is a set of data attributes pertaining
 to something of interest to a business. Dimensions are things like
 "customers", "products", "stores" and "time". For users of Data
 Warehouses, data dimensions are entry points to numeric facts (e.g.
 sale, profit, revenue) that a business wishes to monitor.
 
-Data Modeling - A process used to define and analyze data requirements
+**Data Modeling** - A process used to define and analyze data requirements
 needed to support the business processes within the scope of
 corresponding information systems in organizations.
 
-Data Modeling Standards - The most basic and standard design principles
+**Data Modeling Standards** - The most basic and standard design principles
 for data modeling. These standards must be adhered to when creating new
 models or updating existing models in the OpenedX ecosystem.
 
-IDA - “Independently Deployed Application”. Similar to a microservice or
+**IDA** - “Independently Deployed Application”. Similar to a microservice or
```

Comment:
CUD is defined without quotes around the phrase, so for consistency IDA should be as well.